modules/network/ctrl.lua

```diff
@@ -4,6 +4,29 @@ local me = microexpansion
 local network = me.network
 local access_level = microexpansion.constants.security.access_levels
 
+local ctrl_recipe = nil
+if minetest.get_modpath("mcl_core") then
```

Comment:
I believe this recipe switch among others can also use the `iron_ingot_ingredient` variable.